tests/test_systems.py

```diff
@@ -73,7 +97,7 @@ def test_errors():
             ],
         )
 
-    with pytest.raises(ansunits.UnitSystemError) as e_info:
+    with pytest.raises(UtilError) as e_info:
```

Comment:
what is a `UtilError`?

Reply:
Should it be the full name? `raise UtilError.UNKNOWN_UNITS` or should Utils not have custom errors?

Reply:
I guess I'm just wondering why did you change it from `UnitSystemError`?  It seems fine before.

Reply:
When the `Unit("eon")` is created it throws the error before the UnitSystem. Should I change this? I plan on moving this error to Unit when Utils.py gets broken up.

Reply:
OK, seems like it should be some other name is all then